src/constants.ts

```diff
@@ -12,6 +19,14 @@ export const ORDER_QUOTER_MAPPING: { readonly [key: number]: string } = {
   12341234: "0xbea0901A41177811b099F787D753436b2c47690E",
 };
 
+export const OFFCHAIN_ORDER_VALIDATOR_MAPPING: {
```

Comment:
what is this for? unused

Reply:
ah yea that was removed in latest commit